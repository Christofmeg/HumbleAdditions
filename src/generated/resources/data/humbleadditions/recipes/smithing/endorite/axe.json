{
  "type": "minecraft:smithing",
  "addition": {
    "item": "humbleadditions:endorite_block"
  },
  "base": {
    "item": "humbleadditions:endorium_axe"
  },
  "result": {
    "item": "humbleadditions:endorite_axe"
  }
}