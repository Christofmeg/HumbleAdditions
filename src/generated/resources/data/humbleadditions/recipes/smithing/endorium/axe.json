{
  "type": "minecraft:smithing",
  "addition": {
    "item": "humbleadditions:endorium_block"
  },
  "base": {
    "item": "minecraft:diamond_axe"
  },
  "result": {
    "item": "humbleadditions:endorium_axe"
  }
}