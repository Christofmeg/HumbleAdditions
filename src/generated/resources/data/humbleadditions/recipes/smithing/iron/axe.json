{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:iron_block"
  },
  "base": {
    "item": "minecraft:golden_axe"
  },
  "result": {
    "item": "minecraft:iron_axe"
  }
}