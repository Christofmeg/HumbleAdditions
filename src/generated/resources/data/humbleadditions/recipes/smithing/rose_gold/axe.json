{
  "type": "minecraft:smithing",
  "addition": {
    "item": "humbleadditions:rose_gold_block"
  },
  "base": {
    "item": "minecraft:golden_axe"
  },
  "result": {
    "item": "humbleadditions:rose_gold_axe"
  }
}