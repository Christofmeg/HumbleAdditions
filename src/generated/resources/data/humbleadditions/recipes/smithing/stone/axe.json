{
  "type": "minecraft:smithing",
  "addition": {
    "tag": "minecraft:stone_tool_materials"
  },
  "base": {
    "item": "minecraft:wooden_axe"
  },
  "result": {
    "item": "minecraft:stone_axe"
  }
}