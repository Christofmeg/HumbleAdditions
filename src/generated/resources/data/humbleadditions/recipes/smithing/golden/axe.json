{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:gold_block"
  },
  "base": {
    "item": "minecraft:stone_axe"
  },
  "result": {
    "item": "minecraft:golden_axe"
  }
}